scripts/changelog.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+
+GENERAL = "General"
+SATELLITE = "Satellite"
+STORAGENODE = "Storagenode"
+TEST = "Test"
+UPLINK = "Uplink"
+GITHUB_LINK = "[{0}](https://github.com/storj/storj/commit/{0})"
+
+
+def git_ref_field(from_ref, to_ref):
+    # Execute command to show diff without cherry-picks
+    cmd = "git cherry {} {} -v | grep '^+'".format(from_ref, to_ref)
+    ps = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    output = ps.communicate()[0]
+    return output.decode()
+
+
+def generate_changelog(commits):
+    changelog = "# Changelog\n"
+    section = {SATELLITE: [], STORAGENODE: [], TEST: [], UPLINK: [], GENERAL: []}
```

Comment:
Should we order these lexographical? That would put the General commits to the top :)